docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy a Slack app to Heroku.
+
+Download the Getting Started app and change into its directory:
+
+```shell
+git clone https://github.com/mwbrooks/bolt-js-getting-started-app.git
+cd bolt-js-getting-started-app/
+```
+
+> 💡 You also can follow along with an existing Bolt app and we'll mention any extra steps that are needed.
+
+Now that you have the app, let's prepare it for Heroku.
+
+---
+
+### Prepare the app for Heroku
+
+Heroku is a flexible platform that uses a few conventions to understand how to host your app. In this section, we'll update your app to use those conventions.
+
+**1. Use a Git repository**
+
+Before you can deploy your app to Heroku, you need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), initialize a local Git repository, and commit your code to it.
```

Comment:
I think we can remove most of this section. I'd do something like:

Before you can deploy your app to Heroku, you'll need a git repository. If you aren't already using Git, you'll need to [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instructions to [create a Git repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository).

You can keep the > Skip this step... part, but I'd remove everything below that.

Reply:
Good suggestion! It shortens up this section for the majority of developers. I've applied this in 7ee5c1d